shacl12-test-suite/tests/core/property/memberShape-003.ttl

```diff
@@ -0,0 +1,86 @@
+@prefix dash: <http://datashapes.org/dash#> .
+@prefix ex: <http://datashapes.org/sh/tests/core/property/memberShape-003.test#> .
+@prefix mf: <http://www.w3.org/2001/sw/DataAccess/tests/test-manifest#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix sht: <http://www.w3.org/ns/shacl-test#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+ex:Agenda
+  rdf:type rdfs:Class ;
+  rdf:type sh:NodeShape ;
+  rdfs:label "Agenda" ;
+  sh:property ex:Agenda-speakerOrder ;
+.
+
+ex:Agenda-speakerOrder
+  sh:path ex:speakerOrder ;
+  sh:memberShape [
+      sh:nodeKind sh:IRI ;
+    ] ;
+  sh:maxCount 1 ;
+.
+
+ex:Alice
+  rdf:type ex:Person ;
+  rdfs:label "Alice" ;
+.
+
+ex:Bob
+  rdf:type ex:Person ;
+  rdfs:label "Bob" ;
+.
+
+ex:Person
+  rdf:type rdfs:Class ;
+  rdf:type sh:NodeShape ;
+  rdfs:label "Person" ;
+.
+
+# Agenda with invalid list structure (doesn't terminate with rdf:nil)
```

Comment:
Is the SHACL test suite meant to exercise base matters of RDF syntax?  I would expect this test to fail for non-SHACL reasons related to parsing the graph.